working-migration-guides/README.md

```diff
@@ -0,0 +1,51 @@
+# Bevy's Migration Guide Process
+
+Hi! Did someone add `M-Needs-Migration-Guide` to your PR? If so, you're in the right place.
+Let's talk about how this process works.
+
+When we make breaking changes to Bevy, we need to communicate them to users so their libraries and applications can be moved to the new Bevy version.
+To do this, we write and ship a [migration guide](https://bevyengine.org/learn/migration-guides/introduction/) for every major Bevy version.
+To avoid a crunch at the end of the cycle as we *write* all of these,
+Bevy asks authors (and reviewers) to write a draft migration guide as part of the pull requests that make breaking changes.
+
+## What to put in your draft migration guide
+
+A `template.md` file is provided in `bevyengine/bevy/working-migration-guides`: copy-paste that to get started!
+
+Migration guides are intended to briefly communicate:
+
+- what has been changed?
+- why did we make this breaking change?
+- how can users migrate their existing code?
+
+Draft migration guides *do not need to be polished*: it's okay if you're not a native English speaker or aren't a wordsmith.
+Editing is easy; we just want to have an expert's view on the questions above.
+
+When writing migration guides, prefer terse, technical language, and be sure to include terms that users might search for.
+Migration guides are not read end-to-end: instead, they are navigated via Ctrl+F as the reader follows the compiler errors and bugs.
+
+## Note on the `#[deprecated]` attribute
+
+Rust provides a very helpful [`#[deprecated]` attribute](https://doc.rust-lang.org/reference/attributes/diagnostics.html#the-deprecated-attribute), which is a compiler-aware way to mark a piece of Rust code as obsolete and slated for removal.
+This can be a nice a tool to ease migrations, because it downgrades errors to warnings and makes the migration information available right in the user's IDE.
+
+However, it's not always possible to use this attribute, and Bevy does not consider it to be a substitute to a migration guide entry.
```

Comment:
This seems like a useful but somewhat unrelated aside. Maybe it should be placed at the end? That way, the sections focusing on the actual migration guide process are first, and the text maybe flows a bit better. With the current order, I felt like this was kinda sandwiched in, and I had to read pretty far to find where the guide should actually go.